packages/cardano-services/test/Handle/TypeOrmHandleProvider.test.ts

```diff
@@ -0,0 +1,29 @@
+import { TypeOrmHandleProvider, createDnsResolver, getConnectionConfig, getEntities } from '../../src';
+import { logger } from '@cardano-sdk/util-dev';
+
+describe('TypeOrmHandleProvider', () => {
+  let provider: TypeOrmHandleProvider;
+
+  beforeAll(async () => {
+    const dnsResolver = createDnsResolver({ factor: 1.1, maxRetryTime: 1000 }, logger);
+    const entities = getEntities(['handle']);
+    const connectionConfig$ = getConnectionConfig(dnsResolver, 'test', {
+      postgresConnectionStringStakePool: process.env.POSTGRES_CONNECTION_STRING_STAKE_POOL!
+    });
+
+    provider = new TypeOrmHandleProvider({ connectionConfig$, entities, logger });
+
+    await provider.initialize();
+    await provider.start();
+  });
+
+  afterAll(async () => {
+    jest.restoreAllMocks();
+
+    await provider.shutdown();
+  });
+
```

Comment:
From adahandle docs:
> When querying a Handle, you should never allow an empty string. An empty string will return a valid address, but it is not a valid Handle.
```suggestion
  it('does not resolve an empty string handle');
```

Reply:
8e60fbeea should solve this